Convert helm.sh links for theme into internal links

bridgetkromhout opened the pull request:
Hi, @flynnduism - can you check to see if it's okay to switch to https for these links? Thanks.

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

lucperkins commented:
Actually, these should be relative links, so that they can be covered by the link checkeer

bridgetkromhout commented:
Thanks, @lucperkins! I've converted the helm.sh links I changed here, and others in the theme. They look okay locally - we'll want to make sure nothing breaks in preview. (This is not an exhaustive search of the entire site.)
